projects/rocprofiler-systems/tests/rocprof-sys-testing.cmake

```
@@ -549,7 +549,7 @@ function(ROCPROFILER_SYSTEMS_ADD_TEST)
 
     cmake_parse_arguments(
         TEST
-        "SKIP_BASELINE;SKIP_SAMPLING;SKIP_REWRITE;SKIP_RUNTIME;SKIP_SYS_RUN"
+        "SKIP_BASELINE;SKIP_SAMPLING;SKIP_REWRITE;SKIP_RUNTIME;SKIP_SYS_RUN;SKIP_PUSH_POP_CHECK"
```

Don't forget to remove this now.
```suggestion
        "SKIP_BASELINE;SKIP_SAMPLING;SKIP_REWRITE;SKIP_RUNTIME;SKIP_SYS_RUN"
```

---

Whoops